src/context/cart-context.js

```diff
@@ -0,0 +1,65 @@
+import { useEffect, createContext, useContext, useReducer } from "react";
+import axios from "axios";
+
+const CartContext = createContext();
```

Comment:
in this you are using for taking product data you can give product provider , it creates confusion thinking it is  related for Cart data

Reply:
Actually I am using only 1 provider; product data, cart data etc all will be in this same provider.